pkgs/games/ultrastardx/default.nix

```diff
@@ -7,26 +7,24 @@ let
   sharedLibs = [
     pcre portaudio freetype
     SDL2 SDL2_image SDL2_gfx SDL2_mixer SDL2_net SDL2_ttf
-    sqlite lua zlib libX11 libGLU libGL ffmpeg
+    lua sqlite zlib libX11 libGLU libGL ffmpeg
   ];
 
 in stdenv.mkDerivation rec {
   pname = "ultrastardx";
-  version = "2017.8.0";
+  version = "07012019";
```

Comment:
You're sure about that version string?

Reply:
fixed as per the manual ``` If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". ```

Reply:
I saw `version = "unstable-YYYY-MM-DD";` more often than `pname = "foo-unstable";`. Can you change this?

Also, the commit message still contains the old version number.

Reply:
should be fine now